### PR TITLE
Update Bg translation

### PR DESCRIPTION
**Pull request by NikoVP**

### CODE REVIEW

**Comment by qndel** [comment]
That's the third damn PR you've opened and closed. STOP

**Comment by qndel** [comment]
hit me up on discord when you feel like learning git ;)

**Comment by NikoVP** [comment]
i think I managed to do it properly this time! Thanks for the help.

File: Translations/bg.po [hunk 1]
```diff
@@ -1178,83 +1163,95 @@ msgstr "Използване на подробно регистриране на
...
+"Game selection:\n"
 msgstr ""
 "\n"
-"Настройки на Hellfire:\n"
+"Избор на игра:\n"
+
+#. TRANSLATORS: Commandline Option
+#: Source/diablo.cpp:795
+msgid "Force Shareware mode"
+msgstr "Принуди Пробна версия"
```
Comment (NikoVP) [inline Translations/bg.po:1191]:
```suggestion
msgstr "Принудителен режим Пробна версия"
```

File: Translations/bg.po [hunk 2]
```diff
@@ -6790,23 +6814,23 @@ msgid ""
...
+"Леорик сякаш изпадаше във все по-дълбока лудост. Обвиняваше невинни жители "
 "за изчезването на сина си, като ги осъждаше на ужасяваща екзекуция. По-малко "
 "от половината от нас успяха да се спасят от неговото безумие.\n"
 "\n"
-"Неговите рицарите и свещениците се опитаха да го усмирят, но той се обърна "
-"срещу тях. За жалост, те бяха принудени на цареубийство. С последния си дъх, "
-"Кралят отправи жестоко проклятие към бившите си слуги, кълнейки, се че те ще "
-"му служат вечно в мрака.\n"
+"Рицарите му и придворните свещеници се опитаха да го усмирят, но той се "
+"обърна срещу тях и заа жалост, те бяха принудени на цареубийство. С "
```
Comment (NikoVP) [inline Translations/bg.po:6825]:
```suggestion
"обърна срещу тях и за жалост, те бяха принудени на цареубийство. С "
```

**Comment by AJenbo** [comment]
> i think I managed to do it properly this time! Thanks for the help.

Looks good, merging (what you did) is also acceptable for translations :D

**Pull request merged**